docs/deployment/config.adoc

```diff
@@ -824,6 +824,80 @@ clustered.timeout = PT10S
 distributable.acceptInbound:: If true, accept inbound distributable tasks. Default: `true`.
 clustered.timeout:: The time (in ISO-8601 Period format) the node waits for a job (task or task-info job) to be submitted to another cluster node. Default: `PT5S` (5 seconds).
 
+[#scheduler]
+=== Scheduler
+image:xp-770.svg[XP 7.7.0,opts=inline]
+
+`XP_HOME/config/com.enonic.xp.scheduler.cfg`
+
+Configure default custom jobs to be created for scheduling at instance start. Skip creation if a job with the same name exists already. Only recurring (cron) jobs can be created by the config file. There is no guaranteed order for distributed environment, so the first started node will create its version of a job for the whole cluster.
+
+Properties started with `init-job.<my-job-name>` will describe properties of a job with `<my-job-name>` name.
+
+.Params
+[cols="1,1,3", options="header"]
+|===
+|Property
+|Type
+|Description
+
+|`init-job.<my-job-name>.enabled`
+|required
+|`true` - a created job will be able to be run according to its cron value and timezone. `false` - the job will be created, but never run until it changes.
```

Comment:
Until what changes? the property value? the job?

Reply:
Should probably be "_the job will be created, but never run until this property is set to `true`_"